app/controllers/api_controller.rb

```diff
@@ -222,6 +223,20 @@ def render_include_validation_error(exception)
     )
   end
 
+  def render_not_supported_in_old_version_error
+    render(
+      json: {
+        errors: [{
+          title: 'Not Supported In Old Version Error',
+          detail: "This endpoint is not supported in version v#{api_version} - please change the ACCEPT header to a newer version",
+        }],
+      },
+      # NB: 406 Not Acceptable (The resource identified by the request is only capable of generating response entities
```

Comment:
Don't think this comment is really necessary - the return code seems fine and is pretty much self documenting

Reply:
comment removed